spec/functional/resource/user/useradd_spec.rb

```diff
@@ -77,9 +76,19 @@ def password_should_be_set
     end
   end
 
+  def try_cleanup
+    ['/home/foo', '/home/bar'].each do |f|
+      shell_out("rm", "-r", f)
+    end
+    ['cf-test'].each do |u|
+      shell_out("userdel", "-r", username)
```

Comment:
Yeah, not sure looking at that briefly why it wouldn't run. This is from the "artisanal" freebsd 10.1-release box:
```
-bash-4.3$ userdel
-bash: userdel: command not found
-bash-4.3$ echo $?
127
```